packages/core/src/app/common/utility/isFloatingLabelEnabled.ts

```diff
@@ -0,0 +1,9 @@
+import { CheckoutContextProps } from '../../checkout';
+
+export default function isFloatingLabelEnabled(context: CheckoutContextProps): boolean {
+    return (
+        context.checkoutState.data.getConfig()?.checkoutSettings.features[
+            'CHECKOUT-6879.enable_floating_labels'
+        ] ?? false
+    );
+}
```

Comment:
`isFloatingLabelEnabled()` is a helper function to switch floating lables on/off.